spack/prod/var/spack/environments/prod/spack.yaml

```diff
@@ -0,0 +1,91 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  # add package specs to the `specs` list
+  specs:
+  - cmake
+  - r@4.5.1
+  - gcc@13.4.0
+  - blas
+  - openblas
+  - libmd
+  - libbsd
+  - libedit
+  - bison
+  - python@3.12.9
+  - which
+  - texinfo
+  - libtirpc
+  - icu4c
+#  - openmpi@4.1.8+cuda fabrics=ucx schedulers=slurm
+  - openmpi@5.0.8+cuda fabrics=ucx schedulers=slurm
+  - cuda@11.8.0
+  - cuda@12.9.0
+  - go
+  - mpich@4.3.0+cuda+slurm
+  - apptainer@1.4.1~suid
+  - php
+  - py-notebook ^python@3.12.9
+  view:
+    prod:
+      root: /software/spack/prod/view
+      projections:
+        all: "{name}-{version}-{compiler.name}-{compiler.version}-{hash}"
+  concretizer:
+    unify: when_possible
+  mirrors:
+    cache: file:///software/spack/cache
+  packages:
+    all:
+      target: [x86_64_zen4]
+    gcc:
+      externals:
+      - spec: gcc@11.5.0 languages:='c,c++'
+        prefix: /usr
+        extra_attributes:
+          compilers:
+            c: /usr/bin/gcc
+            cxx: /usr/bin/g++
+  modules:
+    default:
+      enable:
+      - lmod
+      roots:
+        lmod: /software/spack/prod/modules
+      lmod:
+        hierarchy: []
+        core_compilers:
+        - gcc@13.4.0
+        hide_implicits: true
+        hash_length: 0
+        projections:
+          all: '{name}/{version}-{compiler.name}-{compiler.version}'
+          gcc: '{name}/{version}'
+          ncl: '{name}/{version}'
+        cuda:
+          environment:
+            prepend_path:
+              LD_LIBRARY_PATH: '{prefix}/lib64'
+        gcc:
+          environment:
+            prepend_path:
+              LD_LIBRARY_PATH: '{prefix}/lib64'
+        netcdf-c:
+          environment:
+            prepend_path:
+              LD_LIBRARY_PATH: '{prefix}/lib64'
+        netcdf-fortran:
+          environment:
+            prepend_path:
+              LD_LIBRARY_PATH: '{prefix}/lib'
+        netcdf-cxx4:
+          environment:
+            prepend_path:
+              LD_LIBRARY_PATH: '{prefix}/lib64'
```

Comment:
Why is this netcdf-specific stuff needed here?

Reply:
But netcdf isn't listed here? Or is it an implicit dependency of something else?